ios/browser/api/history/brave_history_api.mm

```diff
@@ -0,0 +1,241 @@
+/* Copyright (c) 2021 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "brave/ios/browser/api/history/brave_history_api.h"
+
+#include "base/bind.h"
+#include "base/compiler_specific.h"
+#include "base/containers/adapters.h"
+#include "base/guid.h"
+#include "base/sequenced_task_runner.h"
+#include "base/strings/sys_string_conversions.h"
+#include "base/task/post_task.h"
+#include "base/task/thread_pool.h"
+#include "components/browsing_data/core/browsing_data_utils.h"
+#include "components/history/core/browser/history_service.h"
+#include "components/history/core/browser/history_types.h"
+#include "components/keyed_service/core/service_access_type.h"
+#include "ios/chrome/browser/application_context.h"
```

Comment:
this doesn't appear to be used

Reply:
actually it looks like there are a lot of includes here that aren't needed (and please also remove deps for them)

Reply:
All includes are travesed and unnecessary ones are deleted with their deps

Fix at https://github.com/brave/brave-core/pull/8869/commits/0f093cd6b86016fbeb82f9d01f7e4045b6a9cbdc